### PR TITLE
CSPL_2662 - Troubleshooting documentation

### PR DESCRIPTION
**Pull request by akondur**
Troubleshooting documentation for CR status, AppFramework

### CODE REVIEW

**Review by vivekr-splunk — COMMENTED** [review]
Can you also add pause annotations descriptions

**Comment by akondur** [comment]
> Can you also add pause annotations descriptions

Added documentation for `pause annotations` as well as `container logs`.

File: docs/CustomResources.md [hunk 1]
```diff
@@ -347,6 +347,105 @@ __CPU Throttling__
...
+Here is an example of a standalone with the pause annotation set. In this state, the Splunk Operator requeues the reconcillation without performing any reconcile operations unless the annotatation is removed.
+
+```
+apiVersion: enterprise.splunk.com/v4
+kind: Standalone
+metadata:
+  name: test-only-debug
+  namespace: splunk-operator
+  annotations:
+    standalone.enterprise.splunk.com/paused: "true"
```
Comment (vivekr-splunk) [inline docs/CustomResources.md:399]: example should be 
`standalone.enterprise.splunk.com/paused: ""`

Comment (akondur) [reply docs/CustomResources.md:399]: As discussed, closing this as the value of "true" works as well!

**Pull request merged**